### PR TITLE
Changed quotation (" to ')

### PR DESCRIPTION
**Pull request by annando**
Most parts of this PR only consist of changed quotations. Additionally a local function had been renamed and extracted as a preparation for another PR.

### CODE REVIEW

File: src/Content/Text/BBCode.php [hunk 1]
```diff
@@ -109,7 +109,7 @@ private static function getOldAttachmentData($body)
 	 *
 	 * @param string $body Message body
 	 * @return array
-	 *                     'type' -> Message type ("link", "video", "photo")
+	 *                     'type' -> Message type ('link', 'video', 'photo')
```
Comment (MrPetovan) [inline src/Content/Text/BBCode.php]: It's missing `"audio"` and the quote conversion wasn't warranted.

File: src/Content/Text/BBCode.php [hunk 2]
```diff
@@ -38,7 +38,7 @@ class BBCode extends BaseObject
...
```
Comment (MrPetovan) [inline src/Content/Text/BBCode.php]: This replacement wasn't warranted, double quotes are more suitable for natural language quoting.

Comment (annando) [reply src/Content/Text/BBCode.php]: How is our rule concerning this? By now I'm using ```"``` with SQL conditions and whenever the ```'``` is part of a string.

Comment (MrPetovan) [reply src/Content/Text/BBCode.php]: Yes, but for me comments are excluded from this rule as natural language conventions should apply.

Comment (annando) [reply src/Content/Text/BBCode.php]: Can we write down these rules? I have the feeling as if several coders are having different opinions.

Comment (MrPetovan) [reply src/Content/Text/BBCode.php]: Is it that important for comments?

Comment (MrPetovan) [reply src/Content/Text/BBCode.php]: See https://github.com/friendica/friendica/pull/7746

**Pull request merged**